packages/example-next/chains.ts

```diff
@@ -49,102 +49,67 @@ export function getAddChainParameters(chainId: number): AddEthereumChainParamete
-      (url) => url !== ''
-    ),
-    name: 'Rinkeby',
-  },
-  5: {
-    urls: [process.env.infuraKey ? `https://goerli.infura.io/v3/${process.env.infuraKey}` : ''].filter(
-      (url) => url !== ''
-    ),
-    name: 'Görli',
-  },
-  42: {
-    urls: [process.env.infuraKey ? `https://kovan.infura.io/v3/${process.env.infuraKey}` : ''].filter(
-      (url) => url !== ''
-    ),
-    name: 'Kovan',
-  },
-  // Optimism
   10: {
-    urls: [
-      process.env.infuraKey ? `https://optimism-mainnet.infura.io/v3/${process.env.infuraKey}` : '',
-      'https://mainnet.optimism.io',
-    ].filter((url) => url !== ''),
+    urls: [infuraUrl('optimism-mainnet'), 'https://mainnet.optimism.io'].filter(Boolean),
     name: 'Optimism',
     nativeCurrency: ETH,
     blockExplorerUrls: ['https://optimistic.etherscan.io'],
   },
-  69: {
-    urls: [
-      process.env.infuraKey ? `https://optimism-kovan.infura.io/v3/${process.env.infuraKey}` : '',
-      'https://kovan.optimism.io',
-    ].filter((url) => url !== ''),
-    name: 'Optimism Kovan',
-    nativeCurrency: ETH,
-    blockExplorerUrls: ['https://kovan-optimistic.etherscan.io'],
-  },
-  // Arbitrum
   42161: {
-    urls: [
-      process.env.infuraKey ? `https://arbitrum-mainnet.infura.io/v3/${process.env.infuraKey}` : '',
-      'https://arb1.arbitrum.io/rpc',
-    ].filter((url) => url !== ''),
+    urls: [infuraUrl('arbitrum-mainnet'), 'https://arb1.arbitrum.io/rpc'].filter(Boolean),
     name: 'Arbitrum One',
     nativeCurrency: ETH,
     blockExplorerUrls: ['https://arbiscan.io'],
   },
-  421611: {
-    urls: [
-      process.env.infuraKey ? `https://arbitrum-rinkeby.infura.io/v3/${process.env.infuraKey}` : '',
-      'https://rinkeby.arbitrum.io/rpc',
-    ].filter((url) => url !== ''),
-    name: 'Arbitrum Testnet',
-    nativeCurrency: ETH,
-    blockExplorerUrls: ['https://testnet.arbiscan.io'],
-  },
-  // Polygon
   137: {
-    urls: [
-      process.env.infuraKey ? `https://polygon-mainnet.infura.io/v3/${process.env.infuraKey}` : '',
-      'https://polygon-rpc.com',
-    ].filter((url) => url !== ''),
+    urls: [infuraUrl('polygon-mainnet'), 'https://polygon-rpc.com'].filter(Boolean),
     name: 'Polygon Mainnet',
     nativeCurrency: MATIC,
     blockExplorerUrls: ['https://polygonscan.com'],
   },
-  80001: {
-    urls: [process.env.infuraKey ? `https://polygon-mumbai.infura.io/v3/${process.env.infuraKey}` : ''].filter(
-      (url) => url !== ''
-    ),
-    name: 'Polygon Mumbai',
-    nativeCurrency: MATIC,
-    blockExplorerUrls: ['https://mumbai.polygonscan.com'],
-  },
-  // Celo
   42220: {
     urls: ['https://forno.celo.org'],
     name: 'Celo',
     nativeCurrency: CELO,
     blockExplorerUrls: ['https://explorer.celo.org'],
   },
+}
+
+export const TESTNET_CHAINS: ChainConfig = {
+  5: {
+    urls: [infuraUrl('goerli')].filter(Boolean),
+    name: 'Görli',
+  },
+  69: {
+    urls: [infuraUrl('optimism-kovan'), 'https://kovan.optimism.io'].filter(Boolean),
```

Comment:
kovan and rinkeby are deprecated - you can omit them.

Reply:
Ok! Will do! Didn't want to make too many changes re: examples, but I guess it's fine. Will make sure that's covered in the changelog.